applications/tari_app_grpc/Cargo.toml

```diff
@@ -23,10 +23,10 @@ log = "0.4"
 num-traits = "0.2.15"
 prost = "0.9"
 prost-types = "0.9"
-rand = "0.8"
+rand = "0.8.5"
```

Comment:
What happens if this is changed to 0.7.3?

Reply:
Cargo cannot find matching dependencies then. 
Argon2 uses rand_core 0.6, which is used by rand 0.8. 
The rand_core version used by rand and other libraries needs to match up. 
We need to downgrade argon2 too far back to get it to use rand_core 0.5 which is used by rand 0.7. 

Luckily app_grpc does not use tari_crypto which now requires rand to be 0.7.